vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -0,0 +1,131 @@
+Use `ent COMMAND --help` to see the details for each command.
+```
+~~~~~~~~~~~~~~~~~~~
+ Entando CLI
+~~~~~~~~~~~~~~~~~~~
+
+> Essentials:
+  - Activate using:  ~/.entando/ent/v7.0.0/cli/v7.0.0/activate
+  - Deactivate using: ~/.entando/ent/v7.0.0/cli/v7.0.0/deactivate
+> Syntax:
+ent [command] [subcommand] [name] [flags]
+
+> ent Commands:
+  - app                  => Helper for managing an Entando App
+  - app-info             => Displays information about an Entando App
+  - bundle               => Helper for managing bundle functions. See below for more details
+  - check-env            => Checks the environment for required dependencies and settings
+  - diag                 => Runs diagnostics and aggregates the related info in a .tgz file
+  - ent                  => Helper for managing the local ent installations
+  - ecr                  => Helper for managing the ECR
+  - help                 => Help information
+  - host                 => Helper for managing the system that hosts the quickstart VM
+  - jhipster           (not there)
+  - kubectl              => Helper for using kubectl in ent managed scenarios
+  - node-cmd-wrapper    => wrapper of node commands
+  - pod                  => Displays information related to a set of pods
+  - prj                 => Helper for managing Entando bundle projects
+  - profile              => Helper for managing an Entando App
+  - quickstart           => Helper for installing Entando instances locally
+  - run-tests            => Runs the internal tests
+> Further info about entando:
+  - ~/.entando/ent/v7.0.0/cli/v7.0.0/README.md
+  - https://www.entando.com/
+  - https://developer.entando.com/
+
+> ⚠ RECOMMENDED FIRST STEP ⚠ :
+  - Check the dependencies (ent check-env --help)
+```
+** New for version 7.1 **
+```
+~~~~~~~~~~~~~~~~~~~
+ ent bundle CLI
+~~~~~~~~~~~~~~~~~~~
+VERSION
+  @entando/entando-bundle-cli/0.0.1-ENG-3939-PR-101 darwin-x64 node-v14.19.0
+
+USAGE
+  $ ent bundle [COMMAND]
+
+TOPICS
+  api  Manage API claims
+  mfe  Micro Frontend operations
+  ms   Microservice operations
+  svc  Manage auxiliary services
+
+COMMANDS
+  build        Build bundle components
+  generate-cr  Generate the Entando Custom Resource (CR) for a bundle project
+  help         Display help for ent bundle.
+  info         Show status information for the bundle project
+  init         Perform the scaffolding of a bundle project
+  list         List the available components in the bundle
+  pack         Generate the bundle Docker images
+  publish      Publish bundle Docker images
+  run          Run bundle components
+
+```
+### Check the Environment
+
+The `check-env` command verifies the presence of required dependencies and the appropriate versions for your Entando instance, automatically installing dependencies with developer guidance. 
+```
+ent check-env develop
+```
+
+### Update the CLI
+To update ent to the latest version. Run `ent check-env develop` after all updates to check dependecies.
```

Comment:
at first blush this reads like you accidentally added a period when you should have used a comma. it's awkward to call out one command in the black box but backtick the other - and list it before the command that should be run first. these two commands should always be run together so can be documented as a pair:

To update ent to the latest version and check dependencies:
``` sh
 bash <(curl -L "https://get.entando.org/cli") --update
 ent check-env develop
 ```